deprecation-index.yml

```diff
@@ -40,3 +40,8 @@
     - db_query_updated.php
     - DBQueryStatic.php
     - DBQueryStaticUpdated.php
+'getMock()':
```

Comment:
@shaal, let's change this to `BrowserTestBase::getMock()` and update the rector name below.

Reply:
👍